docs/components/topnav-primary/index.html

```diff
@@ -0,0 +1,87 @@
+---
+title: Top Navigation
+minver: 1.0.0
+also:
+  components/topnav-primary/test.html: Testing - Top Navigation
+  elements/hx-navigation-control: <hx-navigation-control>
+---
+{% extends 'component.njk' %}
+
+{% block page_header %}
+  <p>
+    The {{page.title}} provides access to platform-level services and aggregate information
+     across a user's accounts at Rackspace.
+
+  </p>
+{% endblock %}
+
+  {% block content %}
+
+  <section>
+    <header>
+      <h2 id="icon-badge">Nav Menu</h2>
+    </header>
+
+    <div id="vue-eyebrowDemo">
+      <div id="hxTopnav-primary">
+        <a class="img-left brand-logo" href="#"> <img src="images/helix-logo.svg" alt="Logo" /></a>
+        <div id="right-menu">
+          <div>
+            <a href="#" @click="selectedTag(tab)" :current-tab="currentTab">
+              <hx-icon type="bell"></hx-icon>
+              <p>Notifications</p>
+            </a>
+
+            <a href="#" class="selected">
+              <hx-icon type="ticketing"></hx-icon>
+              <p>Tickets</p>
+            </a>
+
+            <a href="#">
+              <hx-icon type="support"></hx-icon>
+              <p>Support</p>
+            </a>
+
+            <a href="#" class="disabled">
+              <hx-icon type="billing"></hx-icon>
+              <p>Billing</p>
+            </a>
+
+            <a href="#">
+              <hx-icon type="account"></hx-icon>
+              <p>Account</p>
+            </a>
+          </div>
+          <div class="straightLine"></div>
+          <div>
+            <hx-disclosure id="topnav-menu" aria-controls="eyebrowMenuId" role="button" aria-expanded="true" class="disabled">
```

Comment:
Please remove the **selected state** from the user account dropdown:

<img width="184" alt="Screen Shot 2020-09-24 at 12 07 56 PM" src="https://user-images.githubusercontent.com/10120600/94177298-ecb6b800-fe5e-11ea-9cc5-cf5c015aaf7e.png">

Reply:
Changes have been done, please review.